### PR TITLE
stevedore-np: Add version 0.8.0

### PR DESCRIPTION
**Pull request by Ovyerus**
Adds [stevedore](https://github.com/slonopotamus/stevedore) to the bucket.

Pretty much a straight, updated copy of TheRandomLabs/scoop-nonportable#296 (done with the original author's permission).

- [x] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

### CODE REVIEW

File: bucket/stevedore-np.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
+{
+    "version": "0.8.0",
+    "description": "Docker distribution for Windows that Just Works",
```
Comment (issaclin32) [inline bucket/stevedore-np.json]:
```suggestion
    "description": "Docker distribution for Windows that aims to provide a frictionless Docker experience on Windows",
```
Taken from [ReadMe / Overview](https://github.com/slonopotamus/stevedore#overview)

Comment (Ovyerus) [reply bucket/stevedore-np.json]: That ends up making it kinda repeat itself

Comment (issaclin32) [reply bucket/stevedore-np.json]: emm... How about **"Docker distribution ~for Windows~ that aims to provide a frictionless Docker experience on Windows"**. I just thought that `that Just Works` seems vague.

Comment (Ovyerus) [reply bucket/stevedore-np.json]: Yeah that sound's better.

Comment (kotx) [reply bucket/stevedore-np.json]: I originally just copied it from the repo description. Is it okay to change it? @slonopotamus

Comment (slonopotamus) [reply bucket/stevedore-np.json]: Sure

**Pull request merged**